core-bundle/tests/Cache/ContaoCacheWarmerTest.php

```diff
@@ -105,6 +106,24 @@ public function testCreatesTheCacheFolder(): void
             "\$this->arrFields = array (\n  'id' => 'int(10) unsigned NOT NULL auto_increment',\n);",
             file_get_contents(Path::join($this->getTempDir(), 'var/cache/contao/sql/tl_test.php'))
         );
+
+        $expected = <<<'TXT'
+            <?php
+
+            return array (
+              'en' =>
+              array (
+                'default' => true,
+                'tl_test' => true,
+              ),
+            );
+
+            TXT;
+
+        $this->assertSame(
+            preg_replace('~\s+~', '', $expected),
+            preg_replace('~\s+~', '', file_get_contents(Path::join($this->getTempDir(), 'var/cache/contao/config/available-language-files.php')))
```

Comment:
```suggestion
            preg_replace('~\s+~', ' ', $expected),
            preg_replace('~\s+~', ' ', file_get_contents(Path::join($this->getTempDir(), 'var/cache/contao/config/available-language-files.php')))
```
Just a nitpick, feel free to ignore ☺️